azure-resources/KeyVault/vaults/recommendations.yaml

```
@@ -1,21 +1,21 @@
-- description: Key vaults should have soft delete enabled
-  aprlGuid: 1cca00d2-d9ab-8e42-a788-5d40f49405cb
+- description: Azure Key Vault RBAC for Data Plane
+  aprlGuid: da780cfe-f727-4327-ac09-bad4edf31522
   recommendationTypeId: null
-  recommendationControl: Disaster Recovery
+  recommendationControl: Security
   recommendationImpact: High
   recommendationResourceType: Microsoft.KeyVault/vaults
   recommendationMetadataState: Active
   longDescription: |
-    Key Vault's soft-delete feature enables recovery of deleted vaults and objects like keys, secrets, and certificates. When enabled, marked resources are retained for 90 days, allowing for their recovery, essentially undoing deletion.
-  potentialBenefits: Enables recovery of deleted items
-  pgVerified: false
+    Enhances security by integrating with Azure RBAC, replacing the legacy Access Policy model.
```

```suggestion
   Boosts security by integrating with Azure RBAC, replacing Access Policy
```

Needs to be less than 60 characters